spec/async/http/cache/general_spec.rb

```diff
@@ -68,43 +68,113 @@
 				else
 					Protocol::HTTP::Response[200, [['cache-control', 'max-age=1, public'], ['vary', 'user-agent']], ['Hello', ' ', 'World']]
 				end
-				
+
 				if request.head?
 					response.body = Protocol::HTTP::Body::Head.for(response.body)
 				end
-				
+
 				response
 			end
 		end
-		
+
 		let(:user_agents) {[
 			'test-a',
 			'test-b',
 		]}
-		
+
 		it "should cache GET requests" do
 			2.times do
 				user_agents.each do |user_agent|
 					response = subject.get("/", {'user-agent' => user_agent})
 					expect(response.headers['vary']).to include('user-agent')
-					expect(response.read).to be == user_agent
+					expect(response.read).to eq(user_agent)
 				end
 			end
-			
-			expect(store.index.size).to be 2
+
+			expect(store.index.size).to eq(2)
```

Comment:
I don't think `be 2` was correct, because that's checking for identity.

If not `eq(2)` then `be == 2`.

Reply:
Integers are value types so it won't matter, but I'm fine with `== 2`.

Reply:
(Agreed that it will work on that technicality. But it tripped me up to see the explicit identity check because then I wondered why that was important?)